providers/infrastructure-aws/v2.0.2/cconly/overlay-kube-apiserver-admission.yaml

```
@@ -0,0 +1,192 @@
+          template: |-
+            path: /etc/kubernetes/admission-control-config.yaml
+            content: |-
+              {{ $namespace_exemptions := printf "%q, %q" "kube-system" "tkg-system" -}}
+              {{ $defaultWarnAudit := "baseline" }}
+              {{- if .podSecurityStandard.exemptions.namespaces -}}
+                {{ range $namespace := .podSecurityStandard.exemptions.namespaces -}}
+                  {{ $namespace_exemptions = printf "%s, %q" $namespace_exemptions $namespace -}}
+                {{- end -}}
+              {{- end -}}
+              apiVersion: apiserver.config.k8s.io/v1
+              kind: AdmissionConfiguration
+              plugins:
+              {{ if and (not .podSecurityStandard.deactivated) (semverCompare ">= v1.24" .builtin.controlPlane.version) }}
+              - name: PodSecurity
+                configuration:
+                  apiVersion: pod-security.admission.config.k8s.io/v1beta1
+                  kind: PodSecurityConfiguration
+                  defaults:
+                    enforce: "{{ if .podSecurityStandard.enforce -}}
+                        {{ .podSecurityStandard.enforce }}
+                      {{- end }}"
+                    enforce-version: "{{ .podSecurityStandard.enforceVersion -}}"
+                    audit: "{{ if .podSecurityStandard.audit -}}
+                        {{ .podSecurityStandard.audit }}
+                      {{- else -}}
+                        {{ $defaultWarnAudit }}
+                      {{- end }}"
+                    audit-version: "{{ .podSecurityStandard.auditVersion -}}"
+                    warn: "{{ if .podSecurityStandard.warn -}}
+                        {{ .podSecurityStandard.warn }}
+                      {{- else -}}
+                        {{ $defaultWarnAudit }}
+                      {{- end }}"
+                    warn-version: "{{ .podSecurityStandard.warnVersion -}}"
+                  exemptions:
+                    usernames: []
+                    runtimeClasses: []
+                    namespaces: [{{ $namespace_exemptions }}]
+              {{- end }}
+              {{ if .eventRateLimitConf }}
+              - name: EventRateLimit
+                path: eventConfig.yaml
+              {{- end }}
+    enabledIf: '{{ or (and (not .podSecurityStandard.deactivated) (semverCompare ">= v1.24" .builtin.controlPlane.version)) (.eventRateLimitConf) }}'
+    #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: eventRateLimitConf
+  #@overlay/append
+  - name: eventRateLimitConf
+    description: "Adds an admission configuration for EventRateLimit."
+    definitions:
+    - selector:
+        apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+        kind: KubeadmControlPlaneTemplate
+        matchResources:
+          controlPlane: true
+      jsonPatches:
+      - op: add
+        path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraArgs/-"
```

```suggestion
        path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraArgs/enable-admission-plugins"
```

Extra arg is a map, not an array. `/-` to append could only be used for arrays.